presto-docs/src/main/sphinx/index.rst

```diff
@@ -3,19 +3,28 @@ Presto Documentation
 ####################
 
 .. toctree::
+   :hidden:
+
+   develop
```

Comment:
I have changes that are in particular for the developer's guide, do we intend to not have those in the TD docs?